libs/gundeck-types/src/Gundeck/Types/Swagger.hs

```diff
@@ -40,6 +41,12 @@ push = defineModel "Push" $ do
         description "Fallback transport"
         optional
 
+pushList :: Model
```

Comment:
Perhaps you could rename the model to `PushTokenList`? For greppability.

Reply:
👍